repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/image-with-caption.ftl

```diff
@@ -3,11 +3,13 @@
 <#macro imageWithCaption imageWithCaption>
     <@hst.link var="imgLink" hippobean=imageWithCaption/>
     <figure class="nhsuk-image" style="width: 100%">
-        <img class="nhsuk-image__img" src="${imgLink}" alt="${imageWithCaption.description}">
-        <#if imageWithCaption.caption?has_content>
-            <figcaption class="nhsuk-image__caption">
-                ${imageWithCaption.caption}
-            </figcaption>
+        <#if imageWithCaption??>
```

Comment:
Seems like `[INFO] [talledLocalContainer] ==> imageWithCaption.description  [in template "webfile:/freemarker/hee/macros/image-with-caption.ftl" at line 7, column 67]` error has been caused by non-existence of `hippogallery:description` property on `hee:imagesetWithCaption` node [which essentially happens when the Editor uploads an image, but doesn't add a description (as it is optional anyway)] and so I afraid wrapping the block `<#if imageWithCaption??>` may not work in this case.

I think rendering `imageWithCaption.description` with default-value operator (https://freemarker.apache.org/docs/dgui_template_exp.html#dgui_template_exp_missing_default) might help us here:
```
<img class="nhsuk-image__img" src="${imgLink}" alt="${imageWithCaption.description!}">
```
Could we try this ? Thanks!

Reply:
Thanks @smanivasag , updated as per requested